src/agent.spec.ts

```diff
@@ -0,0 +1,549 @@
+import { Finding, ethers, FindingSeverity, FindingType } from "forta-agent";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import { createAddress, createTransactionHash } from "forta-agent-tools/lib/utils";
+import agent from "./agent";
+
+const { handleTransaction } = agent;
+
+import {
+  NETHERMIND_DEPLOYER_ADDRESS,
+  AGENT_REGISTRY_ADDRESS,
+  CREATE_AGENT_ABI,
+  UPDATE_AGENT_ABI,
+  DISABLE_AGENT_ABI,
+} from "./constants";
+
+export const iface = new ethers.utils.Interface([
+  CREATE_AGENT_ABI,
+  UPDATE_AGENT_ABI,
+  DISABLE_AGENT_ABI
+]);
+
+const mockAgentId = 1;
+const mockChainIds = [1, 2];
+const mockMetadata = "metadata";
+const testAddress1 = createAddress("0x0000000000000000000000000000000000000001");
```

Comment:
```suggestion
const testAddress1 = createAddress("0x1");
```
You could actually pass arguments to `createAddress` like this. Otherwise, you're already generating a mocked address by just writing all of the zeroes. 😅